File: 724_Find_Pivot_Index/724_Find_Pivot_Index.rs
fn pivotIndex(nums: Vec<i32>) -> i32 {
    let sum = nums.iter().sum::<i32>();
    let len = nums.len();
    let mut sumL: i32 = 0;

    for i in 0..len {
        if sumL == sum - sumL - nums[i] {
            return i as i32;
        }
        sumL += nums[i];
    }
    -1
}

File: 724_Find_Pivot_Index/724_Find_Pivot_index.cpp
int pivotIndex(vector<int>& nums) {
    int sum=0, sumL=0, i=0;
    for( const int& num : nums ) sum += num;
    for( const int& num : nums ) 
    {
        if (sumL == sum - sumL - num) return i;
        sumL += num
        i++;
    }
    return -1;
}